posthog/models/event/events_query.py

```diff
@@ -173,11 +173,10 @@ def run_events_query(
                 query_result.results[index] = list(result)
                 if distinct_to_person.get(distinct_id):
                     person = distinct_to_person[distinct_id]
-                    properties = person.properties or {}
                     query_result.results[index][column_index] = {
                         "uuid": person.uuid,
                         "created_at": person.created_at,
-                        "properties": {"name": properties.get("name"), "email": properties.get("email")},
+                        "properties": person.properties or {},
```

Comment:
Since person properties is (effectively) unbounded in size... Really we should grab just the properties listed in the settings

That "only" saves us sending it over the wire, I assume it doesn't reduce what we query to load the person properties.

The feature is broken right now, so not blocking but I can see us exploding response payload size here

Reply:
Locally this takes me from around 700kb to 900kb

Reply:
Which made me realise https://github.com/PostHog/posthog/pull/14726

Reply:
This will only lead to a long line of "why don't all my properties show up" support requests... Let's bite the bullet for now and see where this ends up...